### DISCUSSION TITLE
Customizing under docker

### DISCUSSION THREAD

**Discussion by cuneglas** [Forceu/Gokapi#28] [Q&A] [answered]
Thanks for the great work.
How can I customize inside a docker container? The documentation refers to something that I don't have access inside docker.
Thanks for your help

**Comment by Forceu** [comment, marked as answer]
In that case you could mount the` /app/templates` and ` /app/static` folders as a volume and then edit it there. Do you have access to volumes?

Otherwise you need to run the binary or clone this repository and build the Docker image from scratch with the modified files.

**Comment by cuneglas** [comment]
With some delay... I mounted the folder, changed the images inside the /assets folder and restarted, nothing happened. I ran the ls command inside the container to see if the folder was actually mounted, and all the files I have inside the folder showed up (I didn't delete the original files I put a _ before). What am I doing wrong?
Thanks for the help and for this awesome tool

**Reply by Forceu** [reply]
I edited the answer above. Are you using the correct image? Normally there should be no /assets folder.

**Reply by cuneglas** [reply]
I explained myself wrong, I mounted the folder and copied the contents from the source code into the folder and then the files that I wanted to change I kept them but renamed them with an underscore

**Reply by cuneglas** [reply]
With the new mounts it worked. Thanks for the help and thanks again for Gokapi

**Reply by Forceu** [reply]
Great to hear it is working now! :)
